### PR TITLE
[QUAD-176]: Make jsp changes to display buttons based on role.

### PR DESCRIPTION
**Pull request by nischalsamji**

### CODE REVIEW

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/dictionary/DictionaryController.java [hunk 1]
```diff
@@ -61,9 +61,8 @@
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/dictionary/DictionaryController.java:61]: Did you change anything in this file, or is it just reformatted?

File: Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp [hunk 2]
```diff
@@ -1,63 +1,109 @@
...
-	  $.ajax({ url: "${pageContext.servletContext.contextPath}/auth/rest/"+ "${project.projectId}"+ "/conceptcollections.json",
-		  type : "GET",
-          success: function(data){
-              $.each(data, function( index, value ) {
-            	 $( "#projectConceptCollections" ).prepend( "<a href='${pageContext.servletContext.contextPath}/auth/conceptcollections/"+value['id']+"'><i class='fa fa-list-alt'></i> "+value['name']+"</a><br>");
-              });
-          }
-	  });
-  });
+	$(document)
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp]: Pleasee reformat that. It's really difficult to read.

File: Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp [hunk 3]
```diff
@@ -1,63 +1,109 @@
...
+															$(
+																	"#projectConceptCollections")
+																	.prepend(
+																			"<a href='${pageContext.servletContext.contextPath}/auth/conceptcollections/"+value['id']+"'><i class='fa fa-list-alt'></i> "
+																					+ value['name']
+																					+ "</a><br>");
+														});
+									}
+								});
+					});
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp]: did you change anything here?

File: Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp [hunk 4]
```diff
@@ -1,63 +1,109 @@
...
+<a
+	href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">http://quadriga.asu.edu/sites/${project.unixName}</a>
+
 
 <hr>
 
-<a href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}/settings"><i class="fa fa-cog"></i> Public Page Settings</a>
+<a
+	href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}/settings"><i
+	class="fa fa-cog"></i> Public Page Settings</a>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp]: did you change anything in this file?

File: Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp (hunk 3 above)
Comment (nischalsamji) [inline Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp]: Formatted it

File: Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp [hunk 5]
```diff
@@ -3,172 +3,202 @@
...
+					class="fa fa-user-plus" aria-hidden="true"></i> Add</a><br> <a
+					href="${pageContext.servletContext.contextPath}/auth/workbench/${project.projectId}/collaborators/delete"><i
+					class="fa fa-user-times" aria-hidden="true"></i> Remove</a><br> <a
+					href="${pageContext.servletContext.contextPath}/auth/workbench/${project.projectId}/updatecollaborators"><i
+					class="fa fa-users" aria-hidden="true"></i> Update</a>
+				</ul>
+			</div>
+		</c:if>
+	</div>
+</div>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp:206]: Same here, what did you change?

Comment (nischalsamji) [inline Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp:206]: Line 191 and formatted it.

**Comment by jdamerow** [comment]
ok, are there any more changes coming? Should I close the PR, or is this done?

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/dictionary/DictionaryController.java (hunk 1 above)
Comment (nischalsamji) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/dictionary/DictionaryController.java:61]: Just the formatting

**Comment by nischalsamji** [comment]
I made the UI changes, I need to test it once more.

**Comment by nischalsamji** [comment]
Buttons are displayed properly based on the role, the page level access/ authorization needs to be fixed.

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
We just talked about what's missing. I'll close the PR for now.

**Comment by jdamerow** [comment]
ok to test

**Pull request merged**